### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by wvangansbeke** [Britefury/cutmix-semisup-seg#4]
Hi @Britefury,

Thank you for making the code publicly available. 

Can you share the pretrained models for the deeplabv2 architecture (Cutmix, Cutout, VAT and ICT) please?
We are having troubles reproducing the results. 

Thanks in advance.

**Comment by Britefury** [comment]
Hi,

I'm afraid I saved the log files, but not the models.
Let me see what I can do about producing these.

**Comment by wvangansbeke** [comment]
Hi @Britefury,

I'm actually training it myself with a different architecture. I made some hyperparameter changes and the results look better now. 
So it should be fine in the end.

Thanks.

**Comment by Britefury** [comment]
Hey cool! :)

Which architecture have you gone for?

**Comment by wvangansbeke** [comment]
Hi @Britefury,

A deeplabv3 with resnet-50 backbone. A resnet101 takes too long to train for me. I'm getting around 50-56% mIoU for 1% of the labels on PASCAL. This is only for 2 different splits, still want to do a third one. Seems plausible based on the numbers in the paper with a deeplabv3+ and resnet101 backbone I guess. 

Cheers,
Wouter
